hera_cal/io.py

```diff
@@ -678,6 +658,61 @@ def iterate_over_times(self, Nints=1, times=None):
             yield self.read(times=times[i:i + Nints])
 
 
+def read_filter_cache(cache_dir):
+    """
+    Load files from a cache specified by cache_dir.
```

Comment:
I'd give a little more details about when this is used and what these caches contain are. Also, if you talk to @plaplant he'll explain why pickles are a bad format (I forget the reasons precisely, I think they are insecure or platform-dependent or something). Anyway, I'm fine with this for now but maybe make an issue to establish some better (probably h5-based) data format?

Reply:
The current purpose of the cache files is to serve as a temporary disk scratch for filtering matrices to speed up calculations that are being conducted across a night (rather then long term cross-platform storage). The cache keys tend to be fairly complicated too (they often involve all x-values of a filtered data array and the values of weights) so I think pickling might be OK for this but I'm open to what @plaplant  thinks.

Reply:
That said, a more robust format that stored the matrices on disk for the long term across nights would probably have major benefits for the RTP.